Makefile

```diff
@@ -78,6 +78,7 @@ GOLANGCI_LINT_VERSION = "v1.60.1"
 lint: generate vanity-import-check
 	go run github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) version
 	go run github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) run
+	./.github/workflows/scripts/check-clang-format.sh
```

Comment:
Maybe run this before golangci-lint, because clang-format is very fast (~100ms on my machine).

Reply:
restructured workflow and added make target with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/310/commits/dbd69d9fe96a0b2da692aa3227c6006c9a4f0665